esmvalcore/_main.py

```diff
@@ -299,12 +299,12 @@ def __init__(self):
         self.config = Config()
         self.recipes = Recipes()
         self._extra_packages = {}
-        if not list(iter_entry_points('esmvaltool_commands')):
+        if not list(entry_points(group='esmvaltool_commands')):
```

Comment:
We should probably avoid calling `entry_points` twice. Do we really need to turn this into a list? Or would `if not entry_points(...` work?